index.js

```diff
@@ -1,9 +1,10 @@
 /** @babel */
+import fs from 'fs';
 import path from 'path';
 import {CompositeDisposable} from 'atom';
 import {allowUnsafeNewFunction} from 'loophole';
 import setText from 'atom-set-text';
-import pkgDir from 'pkg-dir';
+const helpers = require('atom-linter');
```

Comment:
> Just continue using pkgDir.

Then pkgDIr needs to return null if there is not package.json file. As it is, Atom throws a fatal error if current project does not contain.